vsts/scripts/publishSdksFromStagingToProdStorageAccount.sh

```diff
@@ -108,8 +108,13 @@ function copyPlatformBlobsToProdForDebianFlavor() {
     # Not allowed combinations: 
```

Comment:
could you update this comment in its entirety?

Reply:
Updated, thanks !!